docker/README.md

```diff
@@ -0,0 +1,25 @@
+## Docker
+
+### Use docker compose
+
+- Create config.json
+
+Download conifg.json
```

Comment:
Can you fix this typo before merging? :)

Reply:
got it